src/AddTodoForm.js

```diff
@@ -1,14 +1,38 @@
-import React from "react";
-
-function AddTodoForm() {
-    return (
-      <form>
-        <label htmlFor="todoTitle">Title</label>
-        <input type="text" id="todoTitle" />
-        <button type="submit">Add</button>
-      </form>
-    );
-  }
-  
+import React, { useState } from "react";
+
+function AddTodoForm({ onAddTodo }) {
```

Comment:
We haven't quite hit destructuring in the course yet so you should be passing in `props` here instead of `{ onAddTodo }`

Reply:
okay I'll fix it